test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -607,6 +613,7 @@ public void testContentColumn_contentTypeOverrideAndContentTypeCol()
 
   @Test
   public void testContentColumn_contentTypeOverride() throws Exception {
+    assumeFalse("SQL Server does not support blobs", is(SQLSERVER));
```

Comment:
This test has nothing to do with BLOBs. What we ended up doing for MySQL, which has the same problem, is swizzling "clob" to "text". We could do much the same for "blob" and "clob" with SQL Server, perhaps just to varbinary(max) and varchar(max). That may create some other problems, but it wouldn't lead to skipping tests that don't really care about the type of the columns like this.